test/integration/order.test.js

```diff
@@ -0,0 +1,153 @@
+const { describe, before, test } = require('mocha')
+const { expect } = require('chai')
+const createJWKSMock = require('mock-jwks').default
+const nock = require('nock')
+
+const { createHttpServer } = require('../../app/server')
+const { postOrderRoute, postRecipeRoute } = require('../helper/routeHelper')
+const { seed, cleanup } = require('../seeds/orders')
+const { AUTH_ISSUER, AUTH_AUDIENCE, IDENTITY_SERVICE_HOST, IDENTITY_SERVICE_PORT } = require('../../app/env')
+
+describe('order', function () {
+  describe('valid order', function () {
+    this.timeout(15000)
+    let app
+    let authToken
+    let jwksMock
+
+    before(async function () {
+      await seed()
+      app = await createHttpServer()
+      jwksMock = createJWKSMock(AUTH_ISSUER)
+      jwksMock.start()
+      authToken = jwksMock.token({
+        aud: AUTH_AUDIENCE,
+        iss: AUTH_ISSUER,
+      })
+    })
+
+    after(async function () {
+      await cleanup()
+      await jwksMock.stop()
+    })
+
+    beforeEach(async function () {
+      nock(`http://${IDENTITY_SERVICE_HOST}:${IDENTITY_SERVICE_PORT}`)
+        .persist()
+        .get('/v1/members/foobar3000')
+        .reply(200, {
+          alias: 'foobar3000',
+          address: '5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty',
+        })
+        .get('/v1/members/invalid')
+        .reply(404, {})
+        .get('/v1/members/error')
+        .reply(500, {})
+
+      nock(`http://${IDENTITY_SERVICE_HOST}:${IDENTITY_SERVICE_PORT}`)
+        .persist()
+        .get('/v1/members/supplier3000')
+        .reply(200, {
+          alias: 'supplier3000',
+          address: '4FHold46xGXgs5mUiveU2sbTyGBzmstUspZC92UhjJM694jz',
+        })
+        .get('/v1/members/invalid')
+        .reply(404, {})
+        .get('/v1/members/error')
+        .reply(500, {})
+    })
+
+    afterEach(async function () {
+      nock.cleanAll()
+    })
+
+    test('POST Order - 201', async function () {
```

Comment:
might want a test with multiple items. Especially as you have looping and deduplication logic